src/main/proto/de/cyface/protos/model/measurement.proto

```diff
@@ -584,8 +597,7 @@ message Event {
    *
    * Using `64bit` as the timestamp is absolute and in milliseconds since 1970 and, thus, not in the `integer` range.
    *
-   * The value is encoded in the `uint` ProtoBuf format as we don't expect negative timestamps.
-   * Negative numbers are supported but encoded inefficiently in `uint` this data type, compared to `sint`.
+   * The value is encoded in the `uint` ProtoBuf format as we don't expect (and support) negative absolute timestamps.
    */
   uint64 timestamp = 1;
```

Comment:
For non-`repeated` messages like `Events` we store only absolute `timestamp`s, i.e. we don't support negative numbers like in the `repeated` messages like `LocationRecords` and `Accelerations` where we store timestamp offsets and, thus, have to support negative numbers, too.